iris_grib/_save_rules.py

```diff
@@ -455,7 +526,7 @@ def grid_definition_template_12(cube, grib):
         x_step = step(x_cm, atol=1)
         y_step = step(y_cm, atol=1)
     except ValueError:
-        msg = ('Irregular coordinates not supported for transverse '
+        msg = ('Irregular coordinates not supported for Transverse '
```

Comment:
I'm in two minds about whether this should have the word 'yet' between 'not' and 'supported'.  It sounds more accommodating, but does also imply that we intend to do it some day, and I would imagine that if that is the case, it's a very long way off...

Maybe let's just leave it as it is.

Reply:
@corinnebosley it might be that an irregular Transverse Mercator grid is not supported by GRIB itself.